### DISCUSSION TITLE
How to trigger alert of Superset ？

### DISCUSSION THREAD

**Discussion by LELEFIVE** [apache/superset#16203] [General]
![image](https://user-images.githubusercontent.com/4373954/128986605-3c69d5c5-b860-48ea-b8ee-bd3e81b1aa0a.png)

As shown in the figure above， I configed the alert settings,  but superset didn't output any alert logs and the alert never be triggered.

superset image version 1.0.1

**Comment by LELEFIVE** [comment]
the sql  "select count(*） from benefit_base_info " only have one result.

**Comment by junlincc** [comment]
lots of issues have been fixed in this feature. please check out latest master and let us know if this is still an issue? thanks!
